types/sinonjs__fake-timers/index.d.ts

```diff
@@ -338,8 +339,8 @@ export interface FakeTimerInstallOpts {
     now?: number | Date | undefined;
 
     /**
-     * An array with explicit function names to hijack. When not set, @sinonjs/fake-timers will automatically fake all methods except nextTick
-     * e.g., FakeTimers.install({ toFake: ["setTimeout", "nextTick"]}) will fake only setTimeout and nextTick
+     * An array with names of APIs to hijack. When not set, `@sinonjs/fake-timers` will fake all APIs except `nextTick()` and `queueMicrotask()`.
```

Comment:
nit: "names of APIs to hijack" -> "names of methods to fake" seems more in-line with existing docs?

I mean "APIs" is not necessarily an incorrect term, just unconventional here.

Reply:
Indeed, this is tricky. The `Date` object is also faked. That is why I went for "APIs" instead of "methods/properties/objects". What about: "date, performance, time and timer APIs"? Verbose, not sure if that is more clear. Is there a convention in JS how to call these in a simple way?

Reply:
Agreed. "global methods and APIs" – short and clear.